content/docs/usage.md

```diff
@@ -1,70 +1,100 @@
 # Using CML
 
-You'll need a GitHub or GitLab account to begin. Users may wish to familiarize
-themselves with [Github Actions](https://help.github.com/en/actions) or
-[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration/).
+You'll need a GitLab, GitHub, or Bitbucket account to begin. Users may wish to
+familiarize themselves with [Github Actions](https://help.github.com/en/actions)
+or
+[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration).
```

Comment:
**You** versus **users**... who wins?

Reply:
lol neither would be my choice. "You" is far too informal, while "users" is too disinterestedly cold. Inconsistency is even worse.

```suggestion
A GitLab, GitHub, or Bitbucket account is required. Familiarity with
[Github Actions](https://help.github.com/en/actions) or
[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration)
may also be beneficial.
```